database/migrations/2023_09_16_080713_create_hotels_table.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+use Illuminate\Database\Migrations\Migration;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Support\Facades\Schema;
+
+return new class extends Migration
+{
+    /**
+     * Run the migrations.
+     */
+    public function up(): void
+    {
+        Schema::create('hotels', function (Blueprint $table) {
+            $table->id();
+            $table->unsignedBigInteger('user_id');
+            $table->foreign('user_id')->references('id')->on('users');
+            $table->char('hotel_name');
+            $table->char('description', 255);
+            $table->integer('location');
+            $table->integer('rating');
+            $table->integer('price_range');
+            $table->char('amenities', 255);
+            $table->date('check_in_time');
```

Comment:
replace `$table->date()` with `$table->dateTime()`

Reply:
I have applied the changes